doc/rst/developer/bestPractices/buildingdocs.rst

```diff
@@ -1,15 +1,227 @@
 .. _readme-buildingdocs:
 
+.. _Chapel Developers: https://chapel.discourse.group/c/developers
+
 =============================
 Building Chapel Documentation
 =============================
+The live Chapel web documentation lives at https://chapel-lang.org/docs/.
+All of the source for the documentation is in the chapel source repository, and
+this note talks about how those are organized and built into the html documentation
+seen on the website.
 
-The easiest way to build the Chapel web documentation is to use the top level
-make target::
-
-   [g]make docs
+The easiest way to build the Chapel web documentation is to run the command
+``make docs`` in the top level directory of your ``chapel`` Github clone.
 
 This will build the documentation for the standard and internal modules using
 chpldoc, and symlink all of the ``$CHPL_HOME/doc/`` documents over to
```

Comment:
This isn't your text, but as long as we're cleaning this up:  "standard and internal modules" feels very limited given that we also build the documentation for package modules, the language spec, the "README" style documentation for using Chapel, etc.  So I would either change this to "This will build all of the web-based documentation..." or else "This will build the language spec, module documentation, technical notes, and everything else that ends up in the web documentation."  I also feel like the "symlink all the..." part isn't all that useful since the directory given as the source and destination is the same (so it seems vague / not particularly illuminating).  Of this text, I think that mentioning the use of sphinx and chpldoc in generating these docs is good context, but note that not all the documents use chpldoc, so maybe the replacement for those parts is something like "The docs are built using chpldoc, sphinx, and some other homemade scripts." (?).  I think this paragraph used to be correct and has simply gotten increasingly stale as more and more docs have moved to the web (for example, see https://chapel-lang.org/docs/1.11/index.html).

Reply:
So, looking again, I might write:

> This will build all of the web-based documentation hierarchy including the language spec, module documentation, technical notes, etc. using ``chpldoc``, ``sphinx``, and other Chapel-specific scripts.  Once built, the documentation can be viewed by pointing a web browser at ``$CHPL_HOME/doc/html/index.html``.

Reply:
Used your suggestion.